src/components/MultiHopTrade/components/SharedTradeInput/SharedTradeInput.tsx

```diff
@@ -55,6 +56,7 @@ export const SharedTradeInput: React.FC<SharedTradeInputProps> = ({
     >
       <Center width='inherit' alignItems='flex-end'>
         <Box width='full' maxWidth='500px'>
+          <FoxWifHatBanner />
```

Comment:
Preferably blocking: testing at runtime, the layout shift upon clicking "Claim" is a bit jarring, perhaps we either add it there too, or just add a spacer the same height above it?

Reply:
I've added it to the claims component to avoid the layout shift.